src/keySerializer.js

```diff
@@ -0,0 +1,14 @@
+const { Serializer } = require('./serializer');
+
+class KeySerializer extends Serializer {
+  constructor({ descriptor, keyTransform = key => key } = { descriptor: null, keyTransform: key => key }) {
```

Comment:
what are you doing here, shouln't it be `constructor({ descriptor: null, keyTransform: key => key })`?